### ISSUE TITLE
Heating icon 

### ISSUE THREAD

**Issue by ice2009** [peetereczek/HomeAssistantConfig#3]
Hello it is possible turn on heating icon only when thermostat turn on heater or set another color for icon when heater turn on ? I mean this icon 

![Inked20220924_201826](https://user-images.githubusercontent.com/6351562/192140235-2d90ee65-47c6-4a2a-8524-056c23a16c66.jpg)

**Comment by peetereczek** [comment]
I'm not modifying Nextion stock UI
I also recently switched to https://github.com/joBr99/nspanel-lovelace-ui

**Comment by ice2009** [comment]
@peetereczek  yes but original nextion fw turn on this icon  when heater turn on.

**Comment by ice2009** [comment]
@peetereczek and could you help me discovered it?
